Automattic/wp-calypso
# WPSC: Add Debug Log Buttons

Requires an up-to-date `master` branch of WPSC.

To test:
* The Debug Log card should only be visible if the 'Enable Debugging' toggle is enabled.
* SSH into your wpsandbox, `cd` to `$cache_path` (usually `wp-content/cache/`)
* Watch those cryptic hexadecimal `46ae84b26d.php` log files.
* Click the `Reset Logs` button. The current log file should shrink in size.
* Click the `Disable Logs` button. The Debug Log card should vanish, and the 'Enable Debugging' toggle will be disabled.
* Click the `Disable and Delete Logs` button. The Debug Log card should vanish, and the 'Enable Debugging' toggle will be disabled; furthermore, the current log file will be deleted.

TODO:
* `items` reducer:
  * After disabling, set the previously active log to inactive; tests!
  * Upon enabling debugging: Add new debug log to `items` (needs endpoint change); tests!
  * Add test for `_DELETE`
* 'Delete All' button
* Success notice: Change wording to 'Successfully reset' for active log
